Formula/postgresql.rb

```diff
@@ -36,6 +36,14 @@ class Postgresql < Formula
   on_linux do
     depends_on "linux-pam"
     depends_on "util-linux"
+
+    # configure patch to deal with OpenLDAP 2.5
+    # (revisit on next release)
+    depends_on "autoconf"
```

Comment:
```suggestion
    depends_on "autoconf" => :build
```

Reply:
Argh, I *really* need to get a good night's sleep. :man_facepalming: